packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -142,7 +142,11 @@ function cleanupConfig(config: RunConfig, table: Table): RunConfig {
   return config
 }
 
-function generateIdForRow(row: Row | undefined, table: Table): string {
+function generateIdForRow(
```

Comment:
A personal preference around boolean parameters, it would be better if it was like
```
generateIdForRow(row: Row | undefined, table: Table, opts?: { isLinked?: boolean })
```

Idea being that boolean parameters are confusing when looking at a call of the function, e.g.
`generateIdForRow(row, table, true)` - its not obvious immediately the purpose of the bool parameter, vs `generateIdForRow(row, table { isLinked: true })`

Reply:
Changed, I agree with this approach. I did not want to add many changes on the code and in a lot of places we don't use this approach. But I think we all agree is probably the best approach

Reply:
If there is anywhere you see this approach we should mark them with a comment or something like that and change it - it isn't a great approach and I know there are some old hold outs which could do with being converted.